### PR TITLE
Docs/updated documentation

### LINKED ISSUE #8: Add blog post creation guide
Add a beginner friendly guide that walks the reader through how to create a post that can include:
 - How to clone a the repository
 - How to create a branch
 - How to run the project locally
 - Folder structure of the project
 - Markdown formatting requirements
 - How to create a pull request

### LINKED ISSUE #14: Update Readme

### PR DESCRIPTION
**Pull request by natashaannn**
## Description
Added documentation to guide the creation of posts as a contributor, as well as added templates for events and posts. 

## Steps to test
1. Check that `template.md` is not rendered as a post in both events and posts page.
2. Check that all blog posts and events posts are rendering normally.

## Linked PRs / Issue
CLOSES #8 
CLOSES #14

### CODE REVIEW

File: features/blog/utils/posts.ts [hunk 1]
```diff
@@ -6,11 +6,11 @@ import { PostData } from "../types";
...
 
 export function getSortedPostsData() {
-  // Get file names under /posts
-  const fileNames = fs.readdirSync(postsDirectory);
+  // Get file names under /posts other than template.md
+  const fileNames = fs.readdirSync(postsDirectory).filter(fileName => !fileName.includes("template.md"));
   const allPostsData = fileNames.map((fileName) => {
     // Remove ".md" from file name to get id
-    const id = fileName.replace(/\.md$/, "");
+    const id = fileName.replace(/\.md$/, "") ;
```
Comment (ryanntannn) [inline features/blog/utils/posts.ts:13]: i think that this change is unintended

File: features/blog/utils/posts.ts [hunk 2]
```diff
@@ -6,11 +6,11 @@ import { PostData } from "../types";
 const postsDirectory = path.join(process.cwd(), "content/posts");
 
 export function getSortedPostsData() {
-  // Get file names under /posts
-  const fileNames = fs.readdirSync(postsDirectory);
+  // Get file names under /posts other than template.md
+  const fileNames = fs.readdirSync(postsDirectory).filter(fileName => !fileName.includes("template.md"));
```
Comment (ryanntannn) [inline features/blog/utils/posts.ts:10]: Seems like there is repeated logic in getSortedPostsData and getSortedEventsData. Might want to modularize it for easier maintainability

Comment (ryanntannn) [reply features/blog/utils/posts.ts:10]: Maybe fix in another refactor PR as it is not related to this issue

**Review by ryanntannn — APPROVED** [review]
LGTM readme looks super legit

File: features/blog/utils/posts.ts (hunk 2 above)
Comment (natashaannn) [reply features/blog/utils/posts.ts:11]: Done in this PR!

**Pull request merged**